Automattic/woocommerce-payments
# Save selected date preset into session

Fixes #8972 

#### Changes proposed in this Pull Request
Save the user selected preset into session, so that it persists within the session.

#### Testing instructions
- In Payments > Overview > Payment Activity, select a different date range than default.
- Navigate away from the page
- Return to the Payments Overview and observe that the date range previously selected is used.
- Close and re-open the tab, check that Payments Overview shows the default date range.

-------------------


**Post merge**

Not applicable
